packages/components/atoms/f-image-tile/CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.10.3
+------------------------------
+*March 10, 2022*
+
+### Added
+- checked attribute 
+
+### Updated
```

Comment:
Can be done in another PR but the references to `Updated` in this file should using a `type` from here: https://keepachangelog.com/en/1.0.0/

Reply:
👍  will fix in my next PR, thanks @kevinrodrigues ✨